src/cursor/list_search_indexes_cursor.ts

```diff
@@ -0,0 +1,20 @@
+import type { Collection } from '../collection';
+import type { AggregateOptions } from '../operations/aggregate';
+import { AggregationCursor } from './aggregation_cursor';
+
+/** @public */
+export type ListSearchIndexesOptions = AggregateOptions;
+
+/** @public */
```

Comment:
Are we making an intentional choice to have this and the other types exported in `index.ts` being public even though the APIs using them are internal? Is there any risk that we may need to change these underlying cursors before we officially release?